extensions/ql-vscode/package.json

```diff
@@ -1095,6 +1104,11 @@
           "group": "1_queryHistory@1",
           "when": "viewItem == remoteResultsItem"
         },
+        {
+          "command": "codeQLQueries.runLocalQueryContextInline",
+          "group": "inline",
+          "when": "view == codeQLQueries && viewItem == queryFile && codeQL.currentDatabaseItem"
```

Comment:
Maybe a silly question: do we need `codeQL.currentDatabaseItem`?

If you don't have a current database, the "Run Local Query" command pops up a window and asks you to select one anyway. (I don't think it matters either way, just wondering 🙃)

Reply:
Hm, no reason given. I just thought this was the requirement.

Reply:
We can probably remove it in that case, to slightly simplify the `when` 🤷🏽

Reply:
After discussing this and checking the flow for both cases we decided to keep the db check.